src/graphics/aurora/ttffont.cpp

```diff
@@ -173,6 +186,75 @@ void TTFFont::buildChars(const Common::UString &str) {
 	rebuildPages();
 }
 
+void TTFFont::renderBind(const glm::mat4 &transform) const {
+	glUseProgram(_renderable->getProgram()->glid);
+	_material->bindProgram(_renderable->getProgram(), 1.0f);
+	_material->bindGLState();
+	_renderable->getSurface()->bindProgram(_renderable->getProgram(), &transform);
+	_renderable->getSurface()->bindGLState();
+	_mesh->renderBind();
+
+	/**
```

Comment:
Since that's supposed to be a "normal" multi-line comment, and not a Doxygen comment, change that into just plain `/*`, please.